### PR TITLE
Gives vampire thralls an objective listing their master's name

### LINKED ISSUE #11802: Vampire thralling - Thrall doesn't get note about master
**Problem Description**:
During a vampire round, two people were thralled. Azunja Page and Snaccy Mangle, both thralled by two separate vampires. However they were not alerted to themselves being thralled, nor were they generated any objective / note about who thralled them and who their master is.

**What did you expect to happen**: 
The person to be thralled and carry out the vampires orders.

**What happened instead**:
The person got up and started attacking the vampire instead.

**Why is this bad/What are the consequences:**
It's an issue purely because when a vampire thralls someone and that person is not aware of themselves being thrall, they'll ignore the vampire, call security, even attack the vampire to protect themselves.

**Steps to reproduce the problem**:
1) Make yourself a vampire
2) Get enough blood
3) Thrall someone
4) Check their notes, they got no objective about who their master is.

**When did the problem start happening**:
No idea when it started happening.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
Vampire round, just watching a vampire thrall someone and then the thrall attacked them, calling on security. Completely unaware that they were a thrall.

### PR DESCRIPTION
**Pull request by SteelSlayer**
**What does this PR do:**
Makes it easier for vampire thralls too see who their master is

Fixes #11802

thrallobj
enthrallwarning

**Changelog:**
:cl:
add: Gives vampire thralls an objective, which can be viewed in their notes
tweak: Increases the size of the enthralling message seen by newly created vampire thralls
tweak: Vampire thralls are now stunned briefly (about 3 seconds) upon being enthralled
/:cl:

### CODE REVIEW

**Comment by Couls** [comment]
Maybe also add a small stun so people can stop and read the new text

**Comment by Citinited** [comment]
> Maybe also add a small stun so people can stop and read the new text

Is this really needed? Nowhere else does this happen

**Comment by SteelSlayer** [comment]
I don't really mind the if there's a stun or not, I can go both ways. Though now it should be VERY obvious to the thrall that they're now a thrall. They get stunned, get a big red message in the chat, and not to mention they already get the grey "V" hud icon over themselves

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -356,6 +356,7 @@
 
 	to_chat(H, "<span class='biggerdanger'>You have been Enthralled by [user]. Follow [user.p_their()] every command.</span>")
 	to_chat(user, "<span class='warning'>You have successfully Enthralled [H]. <i>If [H.p_they()] refuse[H.p_s()] to do as you say just adminhelp.</i></span>")
+	H.Weaken(2)
```
Comment (Couls) [inline code/game/gamemodes/vampire/vampire_powers.dm]: nitpick but I mean the Stun() command not the weaken, we don't want to drop them we just want to lock their controls so they can't immediately attack back

**Comment by Couls** [comment]
It’s not really applied anywhere else but that’s usually because thralling is usually much more obvious, shadowling thrall is comprised of multiple stuns and revolutionary thrall literally flashes you @citinited

**Comment by Citinited** [comment]
Yes, but it already displays messages cluing you in to what's happening before you actually become a thrall, and it takes a few seconds for the whole process to conclude. If the victim were paying any amount of attention they'd know something was up.

**Pull request merged**